crates/rolldown_utils/src/mime.rs

```diff
@@ -0,0 +1,25 @@
+pub fn get_data_url_mime_by_extension(extension: &str) -> Option<&'static str> {
```

Comment:
Would you mind checking out if there is ready a solid crate to do this thing?

Reply:
I have used `mime` now. The crate `mime_guess` is used for inferring the mime through extension, and `infer` crate is used for interring the mime through file content (magic data), which is natively implemented by Rust and doesn't require `libmagic`.